semgrep/semgrep/cli.py

```diff
@@ -136,6 +136,14 @@ def cli() -> None:
             "to the number of CPUs on the system."
         ),
     )
+    config.add_argument(
+        "--pattern-regex-all",
```

Comment:
I don't think we should add this

Reply:
Yeah, I'm also confused about this I guess. What's the use-case?

Reply:
This was a request from @clintgibler. His use case is running the regex over all files, or at least not just the file extensions associated with the language. E.g. he wants to search a `Gemfile` or `package.json` via a `semgrep` rule for things like `brakeman` or `express`.

I'm not convinced that `semgrep` is the best avenue for this vs. a simple grep or parsing a JSON file directly. The argument being that we could now use `semgrep` more heavily for code inventory type things, or as a stop-gap solution transitioning from other tools using regex's.

Reply:
Removed!